src/ncp/ncp_base.cpp

```diff
@@ -2595,6 +2600,22 @@ ThreadError NcpBase::GetPropertyHandler_MAC_CNTR(uint8_t header, spinel_prop_key
         value = macCounters->mRxErrOther;
         break;
 
+    case SPINEL_PROP_CNTR_TX_PKT_UNICAST:
+        value = macCounters->mTxUnicast;
```

Comment:
Same suggestion as above: Having the `TX_PKT` case next to other `TX_PKT` and the `RX_PKT` next to other `RX_PKT` ones...